# Phase 6c Wave 5: Bekenstein-Hawking, Ryu-Takayanagi, and the Log Correction Wars

**A non-specialist tour of why log corrections to black-hole entropy matter, and why the leading-order $A/(4 G_N)$ formula cannot be the whole microscopic story.**

Companion notebook to `papers/note_rt_ch_bounds/paper_draft.tex` (a short formalization note).

## A naming caveat up front

This note compares two distinct entropy notions that share the $A/(4 G_N)$ leading term:

- $S_{BH}$ — black-hole *thermodynamic* entropy (Bekenstein-Hawking, Kaul-Majumdar microscopic counting).
- $S_{\rm ent}$ — *entanglement* entropy of a boundary CFT region (Ryu-Takayanagi, Calabrese-Cardy, Casini-Huerta).

In AdS/CFT they are deeply connected (Ryu-Takayanagi *equates* boundary entanglement entropy with bulk minimal-surface area divided by 4G_N), but they are not the same object. Phase 6a Wave 3's Kaul-Majumdar derivation gave a microscopic count of $S_{BH}$ for the project's horizon, with a $-3/2$ log correction. The classical RT *formula* gives just $A/(4 G_N)$. Whether RT is taken as a candidate $S_{BH}$ or as $S_{\rm ent}$, comparing it to Kaul-Majumdar's microscopic count exposes the same structural fact: leading-order $A/(4 G_N)$ alone cannot match the microscopic theory, and the gap in our setup is exactly $(3/2)\log(A/(4 G_N))$.

## The Bekenstein-Hawking starting point

In 1973, Bekenstein argued black holes have entropy. In 1974, Hawking computed it: $S_{BH} = A / (4 G_N)$ (in natural units). One quarter the area, in Planck units. This is the most-cited equation in quantum gravity. It's also the *leading* term: the asymptotic-area-law part. It says nothing about what microscopic theory underlies it.

## The log correction

Different microscopic theories of quantum gravity predict different *subleading* corrections — usually logarithms of the area:

$$S_{BH} = \frac{A}{4 G_N} + \alpha \log\!\left(\frac{A}{4 G_N}\right) + \dots$$

The coefficient $\alpha$ depends on the microscopic theory. It is *not* universal across approaches:

- **Kaul–Majumdar (2000)** counted SU(2)$_k$ Chern-Simons edge states on the horizon: $\alpha = -3/2$.
- **Sen (2012)** computed the same quantity using string-theory loop amplitudes for 4D Schwarzschild: $\alpha = 77/45 \approx +1.71$.

Same physical entropy, two different microscopic predictions, gap of $\approx 3.21$. *Not* a numerical accident — a genuine divergence between theories.

## Ryu-Takayanagi and the missing log

Ryu and Takayanagi (2006) made *boundary entanglement entropy* geometric in AdS/CFT: $S_{\rm ent} = (\text{minimal-surface area})/(4 G_N)$. No log term at leading order. *That cannot be a complete microscopic match* if Kaul-Majumdar's $-3/2$ log correction governs the corresponding $S_{BH}$.

## Universal 2D-CFT entanglement-entropy bound

For 2D conformal field theories, Holzhey–Larsen–Wilczek (1994) and Calabrese–Cardy (2004) provided the *universal* upper bound on entanglement entropy of a finite interval: $S_{\rm ent}(L) \leq (c/3)\log(L/\epsilon)$, where $c$ is the central charge and $\epsilon$ a UV cutoff. Casini–Huerta (2009) gave the rigorous free-QFT specialization. Earlier project documents — and the Lean predicate `H_CasiniHuerta_Bound_Valid` — used "Casini–Huerta" as shorthand; we keep that name for the predicate but cite the universal source correctly here.

## What this note shows

Phase 6a Wave 3 derived the Kaul-Majumdar $-3/2$ coefficient from the project's microscopic horizon boundary condition. This note formalizes the *structural consequence*: classical Ryu-Takayanagi (no log term) and Kaul-Majumdar ($-3/2 \log$) cannot simultaneously describe the same entropy quantity. They agree on a single knife-edge: $A = 4 G_N$ exactly, where $\log(1) = 0$ kills the discrepancy. Anywhere else, they differ by $(3/2)\log(A/(4 G_N))$.

## 1. The two formulas, side by side

Plug in concrete reduced areas $A/(4 G_N) \in \{1/2, 1, 2, 4, 10, 100\}$ and compare:

In [1]:
import os, sys
_HERE = os.getcwd()
_PROJ = _HERE if os.path.basename(_HERE) == 'SK_EFT_Hawking' else os.path.dirname(_HERE)
if _PROJ not in sys.path:
    sys.path.insert(0, _PROJ)

from src.rt_ch_bounds import (
    classical_rt_entropy, kaul_majumdar_entropy, rt_kaul_majumdar_gap,
    sen_4d_log_coeff, SEN_KAUL_MAJUMDAR_COEFF_GAP,
    saturated_ch_entropy,
)
from src.core.visualizations import COLORS, fig_rt_ch_bounds_mtc
import math

G_N = 1.0
print(f'  {"reduced area":>14} | {"classical RT":>14} | {"Kaul-Majumdar":>14} | {"gap":>10}')
print('  ' + '-' * 65)
for r in [0.5, 1.0, 2.0, 4.0, 10.0, 100.0]:
    A = r * 4 * G_N
    rt = classical_rt_entropy(A, G_N)
    km = kaul_majumdar_entropy(A, G_N, c0=0.0)
    gap = rt_kaul_majumdar_gap(A, G_N)
    print(f'  {r:>14.2f} | {rt:>14.4f} | {km:>14.4f} | {gap:>10.4f}')

print()
print('  → They agree only at reduced area = 1, i.e., A = 4 G_N (Planckian scale).')
print('  → For a stellar-mass black hole, reduced area is astronomically large; the')
print('    log correction is small but not zero.')

    reduced area |   classical RT |  Kaul-Majumdar |        gap
  -----------------------------------------------------------------
            0.50 |         0.5000 |         1.5397 |    -1.0397
            1.00 |         1.0000 |         1.0000 |     0.0000
            2.00 |         2.0000 |         0.9603 |     1.0397
            4.00 |         4.0000 |         1.9206 |     2.0794
           10.00 |        10.0000 |         6.5461 |     3.4539
          100.00 |       100.0000 |        93.0922 |     6.9078

  → They agree only at reduced area = 1, i.e., A = 4 G_N (Planckian scale).
  → For a stellar-mass black hole, reduced area is astronomically large; the
    log correction is small but not zero.


## 2. The exact gap at reduced area = 2

The Lean theorem `rt_kaulMajumdar_gap_at_reduced_area_two` says: at $A = 8 G_N$, the gap is *exactly* $(3/2) \log 2 \approx 1.040$. Not approximately. Exactly.

This is the kind of statement formal verification is built for: a clean closed-form identity with no leading-order approximation, machine-checked, surviving any future numerical refactor.

In [2]:
G_N = 1.0
A = 8.0
gap = rt_kaul_majumdar_gap(A, G_N)
expected = 1.5 * math.log(2)
print(f'  At A = 8, G_N = 1 (reduced area 2):')
print(f'    classical RT     = {classical_rt_entropy(A, G_N):.10f}')
print(f'    Kaul-Majumdar    = {kaul_majumdar_entropy(A, G_N, 0):.10f}')
print(f'    gap              = {gap:.10f}')
print(f'    (3/2) log 2       = {expected:.10f}')
print(f'    exactness check  = {abs(gap - expected) < 1e-12}')

  At A = 8, G_N = 1 (reduced area 2):
    classical RT     = 2.0000000000
    Kaul-Majumdar    = 0.9602792292
    gap              = 1.0397207708
    (3/2) log 2       = 1.0397207708
    exactness check  = True


## 3. Why log coefficients aren't universal

Both Kaul-Majumdar and Sen claim to compute *the* universal log coefficient for black-hole entropy. They get different answers. This is not because one of them is wrong — they're computing different things implicitly:

- Kaul-Majumdar: SU(2)$_k$ Chern-Simons edge-mode counting on a 2D horizon.
- Sen: string-theory amplitude calculation for 4D Schwarzschild specifically.

Different *regularizations*, different *boundary conditions*, different *microscopic dictionaries*. The log coefficient $\alpha$ depends on these choices. A theory that gets the leading area term right (BH entropy) can still get the subleading log wrong — and *which* log is right is the very question microscopic theories are trying to answer.

In [3]:
print(f'  Kaul-Majumdar log coefficient   = -3/2 = {-1.5:.4f}')
print(f'  Sen 4D Schwarzschild coefficient = {sen_4d_log_coeff():.4f}    (= 77/45 ≈ +1.711)')
print(f'  gap                              = {SEN_KAUL_MAJUMDAR_COEFF_GAP:.4f}    (= 289/90 ≈ 3.211)')
print()
print('  → Sen and Kaul-Majumdar are not refuting each other; they are answering')
print('    different microscopic questions. The point is: the log coefficient is a')
print('    SIGNATURE of microscopic physics, not a universal constant.')

  Kaul-Majumdar log coefficient   = -3/2 = -1.5000
  Sen 4D Schwarzschild coefficient = 1.7111    (= 77/45 ≈ +1.711)
  gap                              = 3.2111    (= 289/90 ≈ 3.211)

  → Sen and Kaul-Majumdar are not refuting each other; they are answering
    different microscopic questions. The point is: the log coefficient is a
    SIGNATURE of microscopic physics, not a universal constant.


## 4. Universal 2D-CFT entanglement-entropy bound

For 2D conformal field theory, Holzhey–Larsen–Wilczek (1994) and Calabrese–Cardy (2004) proved the universal upper bound on entanglement entropy of an interval of length $L$:
$$S_{\rm ent}(L) \leq \frac{c}{3}\log\!\left(\frac{L}{\epsilon}\right),$$
where $c$ is the CFT's central charge and $\epsilon$ a UV cutoff. The bound is *tight* — saturated by free CFTs (rigorous case treated by Casini–Huerta 2009).

Compute the saturated entropy at $L = 1$ across canonical 2D CFTs:

In [4]:
UV = 1e-3
L = 1.0
for label, c in [
    ('Ising',                 0.5),
    ('Tricritical Ising',     0.7),
    ('3-state Potts',         0.8),
    ('Free boson',            1.0),
]:
    S = saturated_ch_entropy(L, c, UV)
    print(f'  {label:24s}  c = {c:.2f}    S_ent(L=1) saturated = {S:.4f}')

print()
print('  → Higher central charge = larger saturated entropy at the same L.')
print('    The bound is *log* — different microscopic regularizations would all')
print('    have to respect this universal envelope.')

  Ising                     c = 0.50    S_ent(L=1) saturated = 1.1513
  Tricritical Ising         c = 0.70    S_ent(L=1) saturated = 1.6118
  3-state Potts             c = 0.80    S_ent(L=1) saturated = 1.8421
  Free boson                c = 1.00    S_ent(L=1) saturated = 2.3026

  → Higher central charge = larger saturated entropy at the same L.
    The bound is *log* — different microscopic regularizations would all
    have to respect this universal envelope.


## 5. The structural conclusion

Lean theorem `rt_eq_kaulMajumdar_iff_trivial_reduced_area`: classical Ryu-Takayanagi and Kaul-Majumdar agree at *exactly* one point: reduced area = 1, i.e., the Planck scale $A = 4 G_N$. Anywhere else, they disagree.

**This is the project's microscopic claim:** if Phase 6a Wave 3's derivation of the Kaul-Majumdar $-3/2$ from microscopic horizon boundary conditions is correct, then classical Ryu-Takayanagi cannot be a complete description of the entanglement entropy. RT must be augmented with the $-3/2 \log$ correction to be consistent.

**Visualization.**

Left panel: classical RT (linear) vs Kaul-Majumdar (concave from below 1, falling further behind RT as reduced area grows). Both pass through $(1, 1)$. Right panel: Casini-Huerta saturated bound across 2D CFT central charges, on a log-$L$ axis.

In [5]:
# viz-ref: fig_rt_ch_bounds_mtc
fig_rt_ch_bounds_mtc().show()

## 6. Honest scope

**What this note proves.** Given Phase 6a Wave 3's microscopic Kaul-Majumdar derivation, classical Ryu-Takayanagi without log corrections is structurally inconsistent with the project's substrate at every non-trivial reduced area. Seven Lean theorems, zero `sorry`, zero new axioms. Cleanly.

**What this note does not prove.**

1. *The bulk minimal-surface construction.* The Lewkowycz-Maldacena replica trick that geometrizes RT in AdS/CFT is out of scope.
2. *The full CH derivation from modular Hamiltonian.* We take CH as a tracked external hypothesis, not a derived theorem.
3. *AdS/CFT spectrum identification.* We do not say *which* boundary CFT a given horizon MTC corresponds to.
4. *Resolution of Sen vs Kaul-Majumdar.* Both microscopic-counting answers are reported; the project formalizes Kaul-Majumdar but does not adjudicate against Sen.

**Why this matters strategically.** Quantum-gravity model-builders write down classical RT and ignore log corrections all the time. This note is a small but rigorous reminder that the leading-order $A/4 G_N$ formula is provably *not* the full story for the project's substrate — and the discrepancy is exactly $(3/2)\log(A/(4 G_N))$, growing logarithmically with horizon area.

## 7. Where to go next

- **Note:** `papers/note_rt_ch_bounds/paper_draft.tex` — short formalization note (3 pages, 5 bibitems).
- **Lean module:** `lean/SKEFTHawking/RTCasiniHuertaBounds.lean` — 7 theorems.
- **Companion technical notebook:** `Phase6c5_RTCasiniHuerta_Technical.ipynb` — paper-section walkthrough.
- **Phase 6a W3 upstream:** `lean/SKEFTHawking/BHEntropyMicroscopic.lean` — `kaulMajumdarS` definition with the $-3/2$ log coefficient and the Sen-disagreement theorem.
- **Sister wave (W4):** Phase 6c Wave 4 builds on the same horizon substrate but in the Hayden-Preskill information-recovery direction. The two waves are independent slices of the same underlying MTC framework.